### ISSUE TITLE
Read a .rc file!

### ISSUE THREAD

**Issue by rec** [plasma-umass/slipcover#25]
From reading your documentation, you could pretty well read a .coveragerc file as-is, as you've kept the same names (good idea!), though of course many .coveragerc features you don't support.

**Comment by jaltmayerpizzorno** [comment]
Yes, I've wanted to do this, but it's had relatively low priority. PR welcome.

**Comment by jaltmayerpizzorno** [comment]
We now read `pyproject.toml` (and take options from `[tool.slipcover]`)... do we still need this?
